src/test/java/nextstep/subway/acceptance/path/PathAcceptanceTest.java

```diff
@@ -0,0 +1,153 @@
+package nextstep.subway.acceptance.path;
+
+import io.restassured.response.ExtractableResponse;
+import io.restassured.response.Response;
+import nextstep.subway.acceptance.util.CommonAcceptanceTest;
+import nextstep.subway.common.Constant;
+import nextstep.subway.line.presentation.request.AddSectionRequest;
+import nextstep.subway.line.presentation.request.CreateLineRequest;
+import nextstep.subway.line.presentation.response.CreateLineResponse;
+import nextstep.subway.path.presentation.response.FindPathResponse;
+import nextstep.subway.station.presentation.request.CreateStationRequest;
+import nextstep.subway.station.presentation.response.CreateStationResponse;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.http.HttpStatus;
+
+import java.util.stream.Collectors;
+
+import static nextstep.subway.acceptance.line.LineAcceptanceStep.지하철_노선_생성;
+import static nextstep.subway.acceptance.path.PathAcceptanceStep.지하철_최단_경로_조회;
+import static nextstep.subway.acceptance.section.SectionAcceptanceStep.지하철_구간_추가;
+import static nextstep.subway.acceptance.station.StationAcceptanceStep.지하철_역_생성;
+import static org.assertj.core.api.Assertions.assertThat;
+
+@DisplayName("지하철 경로 검색")
+class PathAcceptanceTest extends CommonAcceptanceTest {
+
+    private Long 교대역_ID;
+    private Long 강남역_ID;
+    private Long 양재역_ID;
+    private Long 남부터미널역_ID;
+    private Long 역삼역_ID;
+    private Long 강남구청역_ID;
+    private Long 압구정로데오역_ID;
+    private Long 을지로입구역_ID;
+
+    private Long 이호선_ID;
+    private Long 삼호선_ID;
+    private Long 신분당선_ID;
+    private Long 수인분당선_ID;
+
+    /**
+     * 교대역    --- *2호선* ---   강남역   --- *2호선* ---   역삼역
+     * |                        |
+     * *3호선*                   *신분당선*
+     * |                        |
+     * 남부터미널역  --- *3호선* ---   양재역
+     */
+
+    @BeforeEach
+    protected void setUp() {
+        교대역_ID = 지하철_역_생성(CreateStationRequest.from(Constant.교대역)).as(CreateStationResponse.class).getStationId();
```

Comment:
test fixture 잘 구성해주셨는데, 조금 더 가독성 있게 구성할 수도 있을 것 같아요 😢 

```suggestion
        교대역_ID = 지하철_역_생성됨(교대역);
```